.drone.yml

```diff
@@ -248,7 +248,139 @@ volumes:
 node:
   runner: integration
 ---
+kind: pipeline
```

Comment:
is all of this extra integration tests?

Reply:
is simulating the on chain upgrade by submitting proposal to gov etc.